lib/zhongwen_tools/version.rb

```diff
@@ -1,3 +1,3 @@
 module ZhongwenTools
-  VERSION = '0.19.1'
+  VERSION = '0.19.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.